client/whisper_engine.go

```diff
@@ -13,13 +15,13 @@ const (
 	// This determines how much audio we will be passing to whisper inference.
 	// We will buffer up to (whisperSampleWindowMs - pcmSampleRateMs) of old audio and then add
 	// audioSampleRateMs of new audio onto the end of the buffer for inference
-	whisperSampleWindowMs = 5000 // 5 second sample window
+	whisperSampleWindowMs = 24000 // 5 second sample window
```

Comment:
Could you add some context for why you increased the window size?

Reply:
Okay I am onboard with increasing the sample window size

Reply:
From testing i saw that the model sometimes works better if it has more context. Also i implemented the algorithem in a way that it wants to see a segment finished and only then publishes the text. This can be error prone if it never recognises 2 segments in a signle window.

Reply:
Yeah I really like the way this is implemented right now! Thanks again! I have a pretty big roadmap for features on this if your interested in working on it some more. We can connect on discord or something and chat about it

Reply:
sure hit me up via Brasilikum#2024 on dc